test/integration/typescript-custom-tsconfig/test/index.test.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-env jest */
+
+import { join } from 'path'
+import { nextBuild } from 'next-test-utils'
+
+const appDir = join(__dirname, '..')
+
+const warnMessage = /Using tsconfig file:/
+
+describe('Custom TypeScript Config', () => {
+  it('should warn when using custom typescript path', async () => {
```

Comment:
We also need a test to make sure it doesn't warn when the user did not set the next.config.js property

Reply:
Sure! Added it to the `typescript` test.